src/DataStructures/Tensor/EagerMath/ScaleBy.hpp

```diff
@@ -0,0 +1,28 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines function scale_by
+
+#pragma once
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+
+/*!
+ * \ingroup Functions
+ * \brief Divides the components of a tensor by a scalar
+ *
+ * \returns a tensor of the same type as the input tensor
+ */
+template <typename TensorType>
+TensorType scale_by(TensorType tensor, const DataVector& scale) {
```

Comment:
I also tend to interpret `scale_by` to mean multiplication, hence my complaint.

Another option would be to provide a `scale_by` / `multiply_by` that multiplies, and use `unit_tensor = multiply_by(tensor, 1/magnitude(tensor))` for normalizations. Depends if we prefer a function tailored to normalization or a function that is more generically useful.